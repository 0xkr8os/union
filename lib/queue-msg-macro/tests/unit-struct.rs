use queue_msg_macro::msg_struct;

#[msg_struct]
struct UnitStruct;

fn main() {}
