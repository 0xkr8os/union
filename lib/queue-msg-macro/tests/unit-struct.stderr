error: Only supports Named and Unnamed fields
 --> tests/unit-struct.rs:3:1
  |
3 | #[msg_struct]
  | ^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `msg_struct` (in Nightly builds, run with -Z macro-backtrace for more info)
